src/routes/threads.ts

```diff
@@ -146,11 +147,71 @@ async(req: any, res: Response) => {
   }
 });
 
-router.post("/:id/comments", async(req: any, res: Response) => {
-  res.send(200);
+router.post("/:thread_id/comments", routeProtector,
+[param("thread_id").exists().trim().escape(),
+body("content").exists().trim().escape(),
+body("attachments").custom((value) => {
+  if (!value) {
+    return true;
+  }
+  if (Array.isArray(value)) {
+    return areAllElementsUrls(value);
```

Comment:
Would it be better here to either simply filter out non valid urls, or maybe it would be even better to leave them and then the comment UI can display them as invalid to the user?

Reply:
I think the second option might be better (leave them and let the comment UI display as invalid)